packages/front-end/components/Features/RuleList.tsx

```diff
@@ -65,6 +66,41 @@ export default function RuleList({
     return -1;
   }
 
+  // detect unreachable rules, and get the first rule that is at 100%.
+  let unreachableIndex = null;
+  items.forEach((item, i) => {
+    if (unreachableIndex) return;
+
+    // get the schedules on any of the rules:
+    const upcomingScheduleRule = getUpcomingScheduleRule(item);
+
+    const scheduleCompletedAndDisabled =
+      !upcomingScheduleRule &&
+      item?.scheduleRules?.length &&
+      item.scheduleRules.at(-1)?.timestamp !== null;
+
+    const ruleDisabled =
+      scheduleCompletedAndDisabled ||
+      upcomingScheduleRule?.enabled ||
+      !item.enabled;
+
+    // rollouts and experiments at 100%:
+    if (
+      (item.type === "rollout" || item.type === "experiment") &&
+      item.coverage === 1 &&
+      item.enabled === true &&
+      item.condition === "{}" &&
+      !ruleDisabled
+    ) {
+      unreachableIndex = i + 1;
+    }
+
+    // force rule at 100%: (doesn't have coverage)
+    if (item.type === "force" && item.condition === "{}" && !ruleDisabled) {
+      unreachableIndex = i + 1;
+    }
+  });
```

Comment:
The logic should be in a standalone function in `services/features.ts`.   To make it simpler, I think the function should just take a single rule and return `true` if it covers 100% of people.

```ts
export function isRuleFullyCovered(rule: FeatureRule): boolean {...}
```

And we can add unit tests to `features.test.ts`.

This logic is going to get much more complicated soon (when experiments become the source-of-truth), so having a standalone unit tested function will make it much easier to maintain.